src/dvr/dvr_db.c

```diff
@@ -17,6 +17,8 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#define _ISOC9X_SOURCE
```

Comment:
Why is the llabs check removed from configure in your patches? The 'static inline' version should resolve this llabs issue.